mods/persistence/game/machinery/cryopod.dm

```diff
@@ -28,4 +28,30 @@
 	time_entered = world.time
 
 	SetName("[name] ([occupant])")
-	icon_state = occupied_icon_state
\ No newline at end of file
+	icon_state = occupied_icon_state
+
+/obj/machinery/cryopod/verb/self_eject()
+	set name = "Self-eject Pod"
+	set category = "Object"
+	set src in orange(0)
```

Comment:
Doesn't orange specifically exclude the origin?

Reply:
No, somewhat confusingly. It excludes the Center var from the list, which in this case is the user. This could probably be just range(0), if you'd like me to change it.